### PR TITLE
Fix Acunetix360 parser error

### PR DESCRIPTION
**Pull request by TheocharisPetros**
Currently Acunetix360 report may include/reference multiple CWEs for a single finding.

Based on the above, while importing such kind of report it would fail since only a single CWE can be referenced on DefectDojo.

The parser has been updated in order to split the CWE list and set the CWE reference the first of the list.

### CODE REVIEW

**Comment by StefanFl** [comment]
Hi @TheocharisPetros , thank you for this PR. Please provide a unit test for the list of CWE's as well, so that we have tests for both behaviours.

File: components/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "defectdojo",
-  "version": "2.9.0-dev",
+  "version": "2.9.0",
```
Comment (damiencarol) [inline components/package.json:3]: This should not be in the PR

**Review by damiencarol — CHANGES_REQUESTED** [review]

File: components/package.json (hunk 1 above)
Comment (TheocharisPetros) [reply components/package.json:3]: I will close this PR and create a new one, somehow VS Code rebased the branch with accept both.

**Comment by damiencarol** [comment]
@TheocharisPetros I updated the PR and fixed the conflicts but I see now that we don't have unit tests to test the fix, could you add some?

**Pull request closed without merging**